src/harness/scripts/migrate-0.9.x.sh

```diff
@@ -0,0 +1,174 @@
+#!/bin/bash
```

Comment:
Add the following:

Add the following:

set -o errexit  
set -o pipefail 
set -o nounset

Reply:
done